addons/l10n_be_pos_sale/models/res_company.py

```diff
@@ -0,0 +1,15 @@
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    intracom_tax_ids = fields.One2many(
+        'account.tax', compute='_compute_intracom_tax_ids', help='Intra-Communitary product taxes')
+
+    def _compute_intracom_tax_ids(self):
+        for company in self:
+            intracom_fpos = self.env["account.chart.template"].with_company(
+                company).ref("fiscal_position_template_3", False)
+            company.intracom_tax_ids = intracom_fpos.tax_ids.filtered_domain(
```

Comment:
As said on the task, I still have some doubts about that (we could remove it to cover the case where a previously active intracom tax has been deactivated but is still used on a sale order when we settle it).